stores.js

```diff
@@ -64,6 +64,10 @@ export function setCurrentlyEditing(id) {
 // Although this implementation may appear to ID features in order (1, 2, 3,
 // etc), this is NOT an invariant. Do not assume this; it will not be true as
 // soon as a user deletes or reorders an intervention.
+//
+// NOTE! If you call this twice in a row in a `gjScheme.update` transaction
```

Comment:
... You can guess that this happened to me ;) I don't think we'll be adding many features at a time, but if we do, we could allocate and return multiple IDs from here at a time for convenience